app/views/content/landing/how-to-become-a-teacher/_steps.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="row">
-  <div class="grey col">
+  <div class="col">
     <section class="col col-720 centered">
+
```

Comment:
Missing h2 - should say `Find out more about getting into teaching`

See https://getintoteaching.education.gov.uk/landing/how-to-become-a-teacher-mailing-list

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/ecda207e-17e3-477d-b640-d53893926791)